lighthouse-cli/test/cli/index-test.js

```diff
@@ -67,4 +67,33 @@ describe('CLI Tests', function() {
       assert.equal(ret.status, 1);
     });
   });
+
+  describe('print-config', () => {
+    it('should print the default config and exit immediately after', () => {
+      const ret = spawnSync('node', [indexPath, '--print-config'], {encoding: 'utf8'});
+
+      const config = JSON.parse(ret.stdout);
+      assert.strictEqual(config.settings.output[0], 'html');
```

Comment:
I think this would actually be a good candidate for snapshot testing, but I know you're not a fan :)

Reply:
> good candidate for snapshot testing, but I know you're not a fan

I'd make my opinion more nuanced in that if there's constant snapshot churn they lose efficacy as a unit test because we stop paying attention, but I agree this is a good candidate (default config shouldn't change too much...) and less awkward than this style of checking.

We do lose the immediacy of the 'html' here vs the 'json' in the test below (and `auditMode` true vs false), but that might be ok(ish).

I'll add it.

Reply:
What I'm not a fan of is `expect(result).to.not.be.checked.in.a.ridiculous('manner')` :P:P

Reply:
haha alright, very fair. jest isn't big on the dot chains though! at least not nearly as bad as chai